.github/workflows/update-major-and-minor-version.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright The Notary Project Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Update major and minor tags
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  update-major-minor-tags:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.release.tag_name }}
+      - name: Git config
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+      - name: Tag and push new major and minor versions
+        run: | 
+          VERSION=${{ github.event.release.tag_name }}
+          export MAJOR=$(echo ${VERSION} | cut -d '.' -f 1)
+          export MINOR=${MAJOR}.$(echo ${VERSION} | cut -d '.' -f 2)
+          echo ${VERSION}
+          echo ${MAJOR}
+          echo ${MINOR}
```

Comment:
Since force pushing tag is a critical operation, can you check that `MAJOR` and `MINOR` MUST be non-empty to ensure fail-safety?

Reply:
How about utilizing the checking function in https://github.com/cloudflare/semver_bash/blob/c1133faf0efe17767b654b213f212c326df73fa3/semver.sh#L3C1-L13C2

Reply:
like
```console
~ semverParseInto v1.0.0 MAJOR MINOR PATCH SPECIAL
~ echo $MAJOR
1
~ echo $MINOR
0
```

Reply:
@qweeah This function is written by an organization called `Cloudflare`, that's an extra dependency to our project.

Reply:
we don't need to add dependency, just reuse the code snippet or the regular expression to save some efforts write parsing code.